src/components/Theme/index.css

```diff
@@ -1,13 +1,9 @@
 /**
  * Do not edit directly
- * Generated on Fri, 13 May 2022 03:30:08 GMT
+ * Generated on Sat, 13 Aug 2022 05:46:14 GMT
  */
 
 :root {
```

Comment:
These must have been removed from `tokens.json` as the index file shouldn't have any changes with this update.